client/k8s.go

```diff
@@ -246,21 +251,21 @@ func (k K8sClient) GetCVR(name string) ([]cstorv1.CStorVolumeReplica, error) {
 		klog.Errorf("Error while getting cStor Volume Replica for  %s , couldnot fild any replicas", name)
 	}
 
-	return CStorVolumeReplicas.Items, nil
+	return CStorVolumeReplicas, nil
 }
 
 // NodeForVolume used to get NodeName for the volume from the Kubernetes API
 func (k K8sClient) NodeForVolume(volName string) (string, error) {
 
-	label := cStorTypes.PersistentVolumeLabelKey + "=" + volName
+	label := cstortypes.PersistentVolumeLabelKey + "=" + volName
 
 	podInfo, err := k.K8sCS.CoreV1().Pods("").List(metav1.ListOptions{LabelSelector: label})
 	if err != nil {
 		return "", errors.Wrapf(err, "error while getting target Pod for volume %s", volName)
 	}
 
 	if len(podInfo.Items) != 1 {
-		klog.Errorf("Error invalid number of Pods for volume %s : {%s}", volName, err.Error())
+		klog.Errorf("Error invalid number of Pods %s for volume %s : {%s}", len(podInfo.Items), volName, err.Error())
```

Comment:
```suggestion
		klog.Errorf("Error invalid number of Pods %d for volume %s : {%s}", len(podInfo.Items), volName, err.Error())
```
format specifier should be %d because `len` will return integer